src/index.ts

```diff
@@ -17,17 +15,17 @@ const renderUsers = async (config: ApiConfig) => {
   });
 };
 
-const app = () => {
-  const config = (window as any).__CONFIG__;
-  delete (window as any).__CONFIG__;
-
-  renderUsers(config.api);
+const app = (usersServiceInstance: IUsers) => {
+  renderUsers(usersServiceInstance);
 };
 
 window.onload = (event: Event) => {
-  const logger = new Logger();
+  const ioc = createIoCContainer();
```

Comment:
I would recommend calling createIoCContainer in the root of the file, so you can reuse it in another place and export it to other modules.

Reply:
thanks, moved to the top of the file and make it global